pysts/eval.py

```diff
@@ -46,11 +46,11 @@ def multiclass_accuracy(y, ypred):
     Returns (rawacc, class_acc) where rawacc is the accuracy on the whole set
     and class_acc contains accuracies on all classes respectively
     """
-    result = np.zeros(ypred.shape)
+    result = np.zeros(y.shape)
     clss=y.shape[1]
     class_correct=np.zeros(clss)
     ok=0
-    for row in range(ypred.shape[0]):
+    for row in range(y.shape[0]):
         result[row,np.argmax(ypred[row])]=1
         for cls in range(clss):
             if y[row,cls]==result[row,cls]:
```

Comment:
Not a problem to merge this (I think!), but do you remember the rationale for changing this?